content/en/docs/setup/independent/high-availability.md

```diff
@@ -492,11 +492,10 @@ Before running kubeadm on the other masters, you need to first copy the K8s CA c
 
 1. Follow the steps in the [create ssh access](#create-ssh-access) section, but instead of adding to `etcd0`'s `authorized_keys` file, add them to `master0`.
 1. Once you've done this, run:
-     ```bash
-     scp root@<master0-ip-address>:/etc/kubernetes/pki/* /etc/kubernetes/pki
-     rm apiserver.*
-     ```
-
+   ```bash
+   scp root@<master0-ip-address>:/etc/kubernetes/pki/* /etc/kubernetes/pki
+   rm /etc/kubernetes/pki/apiserver*
+   ```
```

Comment:
The formatting for this appears a bit off in the Netlify preview: https://deploy-preview-8752--kubernetes-io-master-staging.netlify.com/docs/setup/independent/high-availability/#option-2-copy-paste

I suspect it might be the lack of empty lines before and after the code block here.

Reply:
hmm, it looks fine to me in the preview.

Reply:
@neolit123 the 'Option 2: Copy paste' heading is indented compared to the previous heading.

Reply:
@detiber 
looks like a https://github.com/russross/blackfriday issue, because the github markdown is fine.
i don't know how to fix this; possibly some trial-and-error trickery would do.

Reply:
In fact, the GH markdown isn't fine. I suspect a rebase will fix the issue -- there's an extra heading tag (#) at line 500/499. Does not appear to have been introduced by this PR, but does not appear in upstream. Needs rebase in any case.